extensions/amp-analytics/0.1/vendors/atinternet.js

```diff
@@ -16,8 +16,12 @@
 
 export const ATINTERNET_CONFIG = /** @type {!JsonObject} */ ({
   'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+  'vars': {
+    'pixelPath': 'hit.xiti',
+    'domain': '.xiti.com',
+  },
   'requests': {
-    'base': 'https://${log}${domain}/hit.xiti?s=${site}&ts=${timestamp}&r=${screenWidth}x${screenHeight}x${screenColorDepth}&re=${availableScreenWidth}x${availableScreenHeight}',
+    'base': 'https://${log}${domain}/${pixelPath}?s=${site}&ts=${timestamp}&r=${screenWidth}x${screenHeight}x${screenColorDepth}&re=${availableScreenWidth}x${availableScreenHeight}',
```

Comment:
If I understand this correctly, you want all the requests send to your server? Is there any reason to override the default domain? If not, why don't we encode it to the base request instead.

Reply:
The goal was to have a default value for the `${domain}` variable, since it can be customized for some clients, but most of them will use our default domain (xiti.com).
I'm not sure I understood correctly how those default vars work though... Is it for this kind of use case? Same goes for `${pixelPath}`. Thanks.

Reply:
@zhouyx just checking if you saw my answer. Thanks!

Reply:
@BenDz There is an issue with replacing the hostname. As all variables will by default be decoded. We are currently working on a fix to this issue.

Reply:
BTW: There's #19757 to track this issue.

Reply:
@BenDz Sorry for the delay here. The team had some discussion on encoding the request domain. We think putting variables in the request domain is risky. 

Your example with `domain=.xiti.com` works, because `encodeURIComponent('.xiti.com') = 'xiti.com'`. But the request will break if one put `domain=.xiti.com/productPage` because the request is encoded to `xiti.com%2FproductPage`. This type of issue will be very difficult to debug if happen.

#19759 is our attempt to fix this issue. 

I understand that you have the best knowledge on the expecting value to the `${log}` and `${domain}`. If you think our concern won't be an issue with your use case and you want to continue with the current approach. We can definitely unblock this PR. But please document (a comment?) that the value for `${log}` and `${domain}` must equal itself after encoding. 

Let me know if this sounds good, Thank you!

Reply:
Sounds good! Thank you.